client/mass/nav.js

```diff
@@ -371,6 +379,11 @@ function setRenderers(self) {
 		}
 	}
 
+	self.deletePlots = () => {
+		const state = self.app.getState()
+		for (const plot of state.plots) self.app.dispatch({ type: 'plot_delete', id: plot.id })
```

Comment:
Should use subactions here to optimize not having multiple dispatches, and only have one store state write step: `{type: 'app_refresh', subactions: plots.map(plot => ({type: 'plot_delete', id: plot.id}))`.

Reply:
ok